### PR TITLE
update e2e tests with new config format

### LINKED ISSUE #581: end to end tests

### PR DESCRIPTION
**Pull request by stfung77**
Closes #581 
why?

with the new config refactor changes, the e2e tests that run in docker wont work anymore

what?

update the e2e tests to use the new config format

addresses - https://github.com/stellar/java-stellar-anchor-sdk/issues/581

### CODE REVIEW

File: integration-tests/docker-compose-configs/anchor-platform-default-configs/anchor-platform-config.yaml [hunk 1]
```diff
@@ -1,128 +1,358 @@
+version: 1
```
Comment (marcelosalloum) [inline integration-tests/docker-compose-configs/anchor-platform-default-configs/anchor-platform-config.yaml:1]: Is that possible for your docker compose to reuse `example.anchor-config.yaml`? Just wondering if we could get rid of this file.

Comment (marcelosalloum) [reply integration-tests/docker-compose-configs/anchor-platform-default-configs/anchor-platform-config.yaml:1]: Good point! It'd be great to have that file tested in a CI flow but I understand that the docker setup makes it hard.

File: integration-tests/docker-compose-configs/docker-compose.base.yaml [hunk 2]
```diff
@@ -8,6 +8,12 @@ services:
     command: "--sep-server"
     environment:
       - STELLAR_ANCHOR_CONFIG=file:/config/anchor-platform-config.yaml
+      - SECRET_SEP10_SIGNING_SEED=SAX3AH622R2XT6DXWWSRIDCMMUCCMATBZ5U6XKJWDO7M2EJUBFC3AW5X
```
Comment (lijamie98) [inline integration-tests/docker-compose-configs/docker-compose.base.yaml:11]: I think Thibault is going to pick on this pattern. Can we submit an issue for the secrets used in tests?

**Review by lijamie98 — APPROVED** [review]
LGTM. 

Please feel free to merge after the review feedbacks are taken care of.

**Pull request merged**